server/src/main/scala/com/cloudera/livy/server/ApiVersioningSupport.scala

```diff
@@ -0,0 +1,98 @@
+package com.cloudera.livy.server
+
+import javax.servlet.http.HttpServletRequest
+
+import org.scalatra.{NotAcceptable, ScalatraBase}
+
+/**
+  * Livy's servlets can mix-in this trait to get API version support.
```

Comment:
I don't use IntelliJ, but lots of Spark developers do and the Spark style is to use javadoc-style indents. So there must be a setting somewhere.